pkg/utils/kube/ctl.go

```diff
@@ -23,9 +23,9 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/kubectl/pkg/cmd/apply"
+	"k8s.io/kubectl/pkg/cmd/auth"
 	cmdutil "k8s.io/kubectl/pkg/cmd/util"
 	"k8s.io/kubectl/pkg/scheme"
-	"k8s.io/kubernetes/pkg/kubectl/cmd/auth"
```

Comment:
Package has been moved, one less dependency on `k8s.io/kubernetes` 🎉

Reply:
that is awesome!